horreum-java-client/src/test/resources/testcontainers/horreum.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
```

Comment:
I really dislike duplicating the 'main' scripts. Could we use a symlink here?

Reply:
i will clean this up

Reply:
I could not get symlinks to work inside a container, so have used maven-resources-plugin to copy required resources into the target dir for the client tests